orc8r/gateway/c/common/config/test/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-include_directories("/usr/src/googletest/googlemock/include/")
+include_directories("/usr/src/googletest/googlemock/include/" .)
```

Comment:
what does this do?

Reply:
So I learned some things about compiling protobufs in my manual effort to get this working.

Our mconfig.proto protobuf includes common.proto, and so that means that when the compiler gets to mconfig.pb.cc and tries to compile it, includes have to be sucked in that indicate a relative path of `orc8r/protos`.  But the protos aren't at that relative path, I want them instead back at `.../c/ommon/config/test/orc8r/protos`. So what we really want is that the compiler includes the local path (.) in every search it does.

Reply:
This didn't work in CI - aborting test execution by CMake - tossing it in this PR =(